Makefile

```diff
@@ -17,6 +17,10 @@ dev-build: image integration-tests-process-network
 tag:
 	@echo "$(COLLECTOR_TAG)"
 
+.PHONY: builder-tag
+builder-tag:
+	@echo "$(COLLECTOR_BUILDER_TAG)"
```

Comment:
Do we really need a make target for this?

Reply:
the thinking here is to keep the builder tag logic in one place rather than hard coding it in the playbooks or in the OSCI scripts